app/src/main/java/com/bc/gordiansigner/helper/ext/DateExt.kt

```diff
@@ -0,0 +1,17 @@
+package com.bc.gordiansigner.helper.ext
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+const val SIMPLE_DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm"
```

Comment:
I do not prefer this one as package level constant. I think it should be inside specific `object` like `DateTimeConstant` or even in a class like `DateTimeUtils` or something to avoid violating single responsibility principal.